admin/resources/user.js

```diff
@@ -28,7 +27,7 @@ module.exports = {
     edit: {
       before: async (request, response, context) => {
         if (request.method === "post") {
-          throw new ValidationError({
+          throw new AdminJS.ValidationError({
```

Comment:
this is ok but you can also import `{ ValidationError }` from `adminjs`. Only bundle function has to be used via `AdminJS.`

Reply:
Thanks. Restored destructed `ValidationError` from AdminJS.